app/javascript/components/pages/Home.js

```diff
@@ -6,7 +6,7 @@ export default class Home extends Component {
     return (
       <div className= "Home" >
       <h1>Welcome to Ecostrong Dome Homes</h1>
-      <p><h3>Off grid Geodesic dome homes built to withstand CAT 5 Hurricanes.</h3></p>
+      <p><h3>Off grid solar powered Geodesic dome homes.</h3></p>
```

Comment:
Why do you have an h3 nested in a p tag?

Reply:
deleted the p tags